db/migrate/20131008213344_create_automatic_exploitation_runs.rb

```diff
@@ -0,0 +1,11 @@
+class CreateAutomaticExploitationRuns < ActiveRecord::Migration
+  def change
+    create_table :automatic_exploitation_runs do |t|
+      t.integer :workspace_id
+      t.integer :user_id
+      t.integer :match_set_id
+
+      t.timestamps
+    end
```

Comment:
There should be an accompanying `change_table` to add indexes for all the foreign keys.